.circleci/config.yml

```diff
@@ -1,20 +1,131 @@
 version: 2
+
+_defaults: &defaults
```

Comment:
see what we do in angular/angular to make yaml anchors less confusing for newbies:
https://github.com/angular/angular/blob/master/.circleci/config.yml#L11

1) comment what this is
2) use an opaque "anchor_xx" in the syntax location where the name is arbitrary, to make clear which identifier is used later

Reply:
Agree, took me a while to understand the syntax myself.